foo

```diff
@@ -0,0 +1,3 @@
+last 5 versions
+safari >= 7
+not dead
```

Comment:
is this file needed? also, different than `.babelrc` (`not dead`)

Reply:
@)#(@)(#!! Thanks for catching that. Nope! Not needed.